### ISSUE TITLE
Add ROI specific loss function to generator 

### ISSUE THREAD

**Issue by Diogo-Valente2111** [junyanz/pytorch-CycleGAN-and-pix2pix#1637]
I would like to add an additional L1 loss function directly on the region of interest (bounding boxes) during training (Pix2pix) to try to improve clarity for object detection. Is it expected to yield better results? What are the steps that I have to follow to accomplish this?
